src/api/decorators.py

```diff
@@ -1,14 +1,54 @@
+import json
+
 from django.http import JsonResponse
 from django.views.decorators.csrf import csrf_exempt
 
+from cms.models import Region, Language
+
 
 def feedback_handler(func):
+    """
+    Decorator definition for feedback API functions and methods
+
+    :param func: decorated function
+    :type func: func
+
+    :return: return handle_feedback function
+    :rtype: func
```

Comment:
```suggestion
    :type func:  ~collections.abc.Callable

    :return: return handle_feedback function
    :rtype:  ~collections.abc.Callable
```
`func` is not a valid python data type. Since `function` isn't either, I propose to use `Callable` in this case...

Reply:
The first one is still missing

Reply:
Damnit, only grepped for `:rtype: func`

Reply:
Commit is amended.